packages/grafana-ui/src/components/VizTooltip/VizTooltipHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import { css } from '@emotion/css';
+import React, { ReactElement } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+
+import { HeaderLabel } from './HeaderLabel';
+import { VizTooltipHeaderLabelValue } from './VizTooltipHeaderLabelValue';
+import { LabelValue } from './types';
+
+interface Props {
+  headerLabel: LabelValue;
+  keyValuePairs?: LabelValue[];
+  customValueDisplay?: ReactElement | null;
+}
+export const VizTooltipHeader = ({ headerLabel, keyValuePairs, customValueDisplay }: Props) => {
+  const styles = useStyles2(getStyles);
+
+  const renderKeyValue = () => {
```

Comment:
Maybe just `renderValue` so it's clearer that value that is rendered is not necessarily a list of key / value pairs?

```suggestion
  const renderValue = () => {
```